package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-find-rules",
-  "version": "3.1.0",
+  "version": "3.1.1",
```

Comment:
squashmerge and rebasemerge ruins the commit history imo; i'd def prefer extra approvals for release commits, and i'd prefer to rebase the PRs themselves and then normal-merge-commit them.